FreeRTOS/FreeRTOS
# Fix compiler warning in MessageBufferDemo.c

Description
-----------
prvReceiverTask() in MessageBufferDemo.c declared a variable as a "MessageBufferHandle_t *" instead of just a "MessageBuffer_t".  This came to light when updating the kernel submodule pointer as the underlying datatype changed in the kernel from a void *, and is fixed by this pull request.

Test Steps
-----------
Build MessageBufferDemo.c with arm-none-eabi-gcc with the warning level set to -Wall -Wextra to ensure there a no compiler warnings.

Related Issue
-----------
None


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
